test/routes-acl.js

```diff
@@ -0,0 +1,25 @@
+/* jshint camelcase: false */
+
+var chai = require('chai');
+chai.should();
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+var app = require('../server/server');
+var request = require('supertest')('https://localhost:3001');
+
+var TOKEN_ENDPOINT = '/oauth/token';
+var CLIENT_ID = '123';
+var CLIENT_SECRET = 'secret';
+
+describe.only('Route based ACLs', function() {
+
+  before(require('./start-server'));
+
+  after(function(done) {
+    app.close(done);
+  });
+
+  it('should ...', function() {
```

Comment:
What does an example test look like? Should I be making a request to `/api/notes` and making sure the ACL is triggered?